libobs-opengl/gl-x11-egl.c

```diff
@@ -506,10 +502,8 @@ static bool gl_x11_egl_platform_init_swapchain(struct gs_swap_chain *swap)
 		goto fail_window_surface;
 	}
 
-	swap->wi->config = plat->config;
 	swap->wi->window = wid;
 	swap->wi->surface = surface;
-	swap->wi->screen = screen_num;
```

Comment:
Maybe we can also remove `screen_num =  get_screen_num_from_root(xcb_conn, geometry->root);` located above. Hence, I set draft.